hw_29_drinks/js/script.js

```diff
@@ -0,0 +1,25 @@
+// Отримати елементи списку li. Потім перетворити їх на масив з тексту кожного елемента(li.textContent);
+// Залишити унікальні рядки у цьому масиві та відсортувати його.
+// // Результат(li елементи) вивести в той же список ul
+
+let list = document.querySelector('#drinks-list');
+
+function displayUniqueSorted(ul) {
+
+    let arrUniqueSorted = Array.from(ul.children).map(liElem =>
+        liElem.textContent
+    ).filter((element, index, array) =>
+        array.indexOf(element) === index
+    ).sort();
```

Comment:
wrong code formatting. Your code should look like
```javascript
let arrUniqueSorted = Array.from(ul.children)
    .map((liElem) => liElem.textContent)
    .filter((element, index, array) => array.indexOf(element) === index)
    .sort();
```
Use Prettier extension for VSCode  https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode